rplugin/python3/nvim_ipy/__init__.py

```diff
@@ -243,8 +243,15 @@ def connect(self, argv):
 
         reply = self.waitfor(self.kc.kernel_info())
         c = reply['content']
-        lang = c['language_info']['name']
```

Comment:
Use `c['language_info'].get('name', 'unknown')`

Reply:
Yep, silly of me. Fixed :) Thanks!